Corona-Warn-App/src/main/res/drawable/ic_info.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="21"
     android:viewportHeight="21">
     <path
-        android:fillColor="#747576"
+        android:fillColor="#007FAD"
```

Comment:
The dark mode color is too dark, should be in color_secondary style

Reply:
Yes, you're right. I just noticed that I've accidentally changed the color on the info button for the statistics cards too.